src/Og.php

```diff
@@ -269,6 +269,34 @@ public static function isGroupContent($entity_type_id, $bundle_id) {
     return \Drupal::service('og.group_audience_helper')->hasGroupAudienceField($entity_type_id, $bundle_id);
   }
 
+  /**
+   * Check if the given entity is a group content.
+   *
+   * This works by checking if the bundle has at least one group audience field
+   * filled.
+   *
+   * @param object $entity
+   *   The entity object.
+   *
+   * @return array
+   *   List of groups where the entity belongs.
+   */
+  public static function getEntityGroups($entity) {
+    $can_be_group_content = \Drupal::service('og.group_audience_helper')->hasGroupAudienceField($entity->type, $entity->bundle);
```

Comment:
You could save the service into a variable to avoid call it twice. `\Drupal::service('og.group_audience_helper')`